lang/java/reef-common/src/main/java/org/apache/reef/runtime/common/driver/evaluator/EvaluatorManager.java

```diff
@@ -306,6 +306,7 @@ public void onEvaluatorException(final EvaluatorException exception) {
           this.messageDispatcher.onDriverRestartEvaluatorFailed(failedEvaluator);
```

Comment:
Because I'm not familiar with the driver restart logic, not sure whether we should do `this.messageDispatcher.close();` here as well. Can someone verify this?